guides/common/modules/proc_synchronizing-python-repositories.adoc

```diff
@@ -13,17 +13,16 @@ You can view the Python packages in the {ProjectWebUI} at *Content* > *Content T
 +
 The *Label* field is populated automatically based on the name.
 . From the *Type* list, select *python*.
```

Comment:
Are you sure about the capitalization of "python"?

Reply:
Yes, the web UI's dropdown list says "python".